docs/cimpl6ToolingReference.md

```diff
@@ -469,7 +468,7 @@ The content of the _/out_ directory depends on which exporters were selected to
 * _cimcore_ - this directory is only used in the process of building the `modeldoc` export, and may not appear in future releases.
```

Comment:
Cimcore is no longer supported. This can be removed.

Reply:
OK, will remove all references to cimcore